docs/data/base/components/click-away-listener/click-away-listener.md

```diff
@@ -1,58 +1,64 @@
 ---
 product: base
-title: Detect click outside React component
```

Comment:
This title was on purpose for SEO. See https://stackoverflow.com/questions/32553158/detect-click-outside-react-component for why